packages/odyssey-react-labs/src/datePickerTheme.tsx

```diff
@@ -0,0 +1,338 @@
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { CSSInterpolation, Theme, ThemeOptions } from "@mui/material/styles";
+import {
+  ArrowLeftIcon,
+  ArrowRightIcon,
+  CalendarIcon,
+  ChevronDownIcon,
+} from "@okta/odyssey-react-mui";
+
+const popupSpacingValue = 5;
+
+type ThemeStyles = ({
+  theme,
+}: {
+  theme: Omit<Theme, "components">;
+}) => CSSInterpolation;
+
+type StateStyles = Record<string, ThemeStyles>;
+
+const dateStyles: StateStyles = {
+  default: ({ theme }) => ({
+    color: theme.palette.text.primary,
+  }),
+  disabled: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.disabled,
+  }),
+  hover: ({ theme }) => ({
+    backgroundColor: theme.palette.grey[100],
+  }),
+  hoverSelected: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.dark,
+    color: theme.palette.primary.contrastText,
+  }),
+  outsideOfMonth: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.text.secondary,
+  }),
+  selected: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.main,
+    color: theme.palette.primary.contrastText,
+  }),
+  today: ({ theme }) => ({
+    backgroundColor: "transparent",
+    color: theme.palette.primary.main,
+  }),
+};
+
+const todayDotStyles: StateStyles = {
+  default: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.main,
+    borderRadius: "50%",
+    content: '" "',
+    height: `${(2 / 16) * (16 / 14)}rem`,
+    position: "absolute",
+    bottom: theme.spacing(1),
+    width: `${(2 / 16) * (16 / 14)}rem`,
+    transform: "translateY(-50%)",
+  }),
+  hover: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.main,
+  }),
+  selected: ({ theme }) => ({
+    backgroundColor: theme.palette.primary.contrastText,
+  }),
+};
+
+export const datePickerTheme: ThemeOptions = {
+  components: {
+    MuiCalendarPicker: {
+      styleOverrides: {
+        root: ({ theme }) => ({
+          paddingBottom: theme.spacing(popupSpacingValue),
+          paddingLeft: theme.spacing(popupSpacingValue),
+          paddingRight: theme.spacing(popupSpacingValue),
+          paddingTop: theme.spacing(4),
+          width: "100%",
+
+          "&, &::before, &::after": {
+            boxSizing: "border-box", // TEMP. Remove this when scoped CSS is added.
+          },
+        }),
+      },
+    },
+    MuiCalendarOrClockPicker: {
+      styleOverrides: {
+        root: () => ({
+          "& > div": {
+            width: `${(296 / 16) * (16 / 14)}rem`,
```

Comment:
Will these every vary as designs change? (296, 214, etc)? Do they need to be captured somewhere?

Reply:
Also feeling iffy about this since `296` is a magic number. If there's no good fluid way to determine the appropriate width, we could at least change `14` to `theme.typography.fontSize`

Reply:
They're specific to this set of styles. I can't _not_ have it. I'd prefer to have it be `"auto"`, but MUI expects a fixed-width here for whatever reason.

It's also possible I can remove this. I was having border-box problems, but I think I made this change prior to that addition.

I'll re-check and see if I can safely remove this fixed-width because YUCK!

Also, I think this number is calculated off of the width of the labels * 7 + padding * 2. I can probably make it not a magic number regardless.